compiler-rs/clients_schema_to_openapi/src/paths.rs

```diff
@@ -311,6 +313,25 @@ fn get_path_parameters(template: &str) -> Vec<&str> {
     result
 }
 
+fn split_summary_desc(desc: String) -> SplitDesc{
```

Comment:
Nit: this function doesn't need ownership of its parameter as it will create new strings. Changing the parameter to `&str` will avoid the caller to `clone()` the parameter.